### DISCUSSION TITLE
Why is OpenSnitch not honoring priority?

### DISCUSSION THREAD

**Discussion by MrCybertux** [evilsocket/opensnitch#1111] [Q&A]
I wanted to deny traffic on port 1080 to all apps witch do not have the allowance therefore i created 2 rules:

![image](https://github.com/evilsocket/opensnitch/assets/67368875/2e64f690-42a6-4f5c-9e9d-5056bb47622b)
![image](https://github.com/evilsocket/opensnitch/assets/67368875/23c6d955-a4c6-45ea-87ed-6c29a0256995)

From Pfsense i am used that lower nummber means higher priority 
So normaly deny all is the lowest rule and the allow rules should be above that
But all connections from firefox are blocked:

![image](https://github.com/evilsocket/opensnitch/assets/67368875/51bb0f86-fcc7-4612-b321-b8d86d2a9390)

Could someone explain to me why and help me out getting it right?

**Comment by gustavo-iniguez-goya** [comment]
hi @MrCybertux !

Mark `[x] Priority rule` on the Allow rule, otherwise it'll keep evaluating rules until it finds a Deny rule.
